examples/src/bin/playbin.rs

```diff
@@ -9,10 +9,11 @@ fn main() {
     gst::init().unwrap();
 
     let args: Vec<_> = env::args().collect();
-    let uri: &str = if args.len() == 2 {
+    let uri: &str = if args.len() >= 2 {
```

Comment:
If you provide more than 2, it will ignore all others. Shouldn't it rather fail and say that only one argument should be provided.

Reply:
hmm.. true

Reply:
fixed